components/KnowledgePanel.css

```diff
@@ -5,3 +5,15 @@
     width: -moz-fit-content;
     padding: 8px;
 }
+

```

Comment:
Oh I didn't even notice, but these changes should be in `Header.css` rather than `KnowledgePanel.css` just for organization (again doesn't really matter but just to be consistent). Then you can import it in `Header.tsx` using `import './Header.css'`